com.vrcfury.vrcfury/Editor/VF/Inspector/VRCFuryHapticSocketEditor.cs

```diff
@@ -277,7 +290,8 @@ public class VRCFuryHapticPlaySocketEditor : UnityEditor.Editor {
                     var mainLight = main.AddComponent<Light>();
                     mainLight.type = LightType.Point;
                     mainLight.color = Color.black;
-                    mainLight.range = lightType == VRCFuryHapticSocket.AddLight.Ring ? 0.4202f : 0.4102f;
+                    mainLight.range = lightType == VRCFuryHapticSocket.AddLight.Ring ? 0.4202f :
+                        lightType == VRCFuryHapticSocket.AddLight.RingBidirectional ? 0.4201f : 0.4102f;

```

Comment:
I'm unsure as to how the SPSLL subtype and the bidirectional subtype should interact, maybe treat them as flags?